### PR TITLE
Fix for issue #210

### PR DESCRIPTION
**Pull request by jtkDvlp**
Fix for issue #210 that works for me. 

`$(this).load` doesn't work without args!

It´s not necessary to call `load` because either it is already loaded or it will be and the pre given handler will work.

Hope it helps.

### CODE REVIEW

**Comment by jtkDvlp** [comment]
Help please, I dont know why it has failed.

File: js/parallax.js [hunk 1]
```diff
@@ -39,7 +39,7 @@
         $this.children("img").one("load", function() {
           updateParallax(true);
         }).each(function() {
-          if(this.complete) $(this).load();
+          if(this.complete) updateParallax(true);
```
Comment (cerebroso) [inline js/parallax.js]: what if you change 
```
trigger('load');
```
to
```
trigger('load')
```
?
That's what the jquery 3 migration guide recommends

https://jquery.com/upgrade-guide/3.0/#breaking-change-load-unload-and-error-removed

**Comment by NitroBAY** [comment]
any news ?

File: js/parallax.js (hunk 1 above)
Comment (jtkDvlp) [inline js/parallax.js]: Sry, I missed the comment. I will test it myself and will give feedback and accordingly a commit.

**Comment by jtkDvlp** [comment]
Do I have to reslove the merge conflict? oO

**Comment by tomscholz** [comment]
This is fixed

**Pull request closed without merging**